scrapper.py

```diff
@@ -21,41 +33,91 @@ class IncorrectNumberOfArticlesError(Exception):
     """
 
 
+class BadStatusCode(Exception):
+    """
+    Custom error
+    """
+
+
 class Crawler:
     """
     Crawler implementation
     """
+
     def __init__(self, seed_urls, max_articles: int):
-        pass
+        self.seed_urls = seed_urls
+        self.total_max_articles = max_articles
+        self.urls = []
+
+    @staticmethod
+    def _modify_link(link):
+        if 'https://alp.iling.spb.ru/' not in link:
+            link = 'https://alp.iling.spb.ru/' + link[2:]
+        return link
 
     def _extract_url(self, article_bs):
-        pass
+        contents_bs = article_bs.select('div.content-cont-col-text a')
+        links = [self._modify_link(article['href']) for article in contents_bs]
+        return links
```

Comment:
is there a way to extract one url per call as the method name tells us it should get only one link per call?